CDP4IMEInstaller/ThirdPartyLicense/Microsoft.NET.Test.Sdk.txt

```diff
@@ -0,0 +1,68 @@
+﻿name: Microsoft.NET.Test.Sdk
```

Comment:
the test related libs do not need to be here as they are not being distributed with the installer, please remove